e2e/wdio/core/tests/wizard.e2e-spec.ts

```diff
@@ -0,0 +1,232 @@
+        expect(getElementClass(mobileExample + step, 1)).toContain('step--current', 'button did not direct you to the step 2');
+    });
+
+    it('should check compliting form for mobile example', () => {
+        const stepsLength = getElementArrayLength(mobileExample + step) / 2;
+        for (let i = 0; i < stepsLength; i++) {
+            expect(getElementClass(mobileExample + step, i)).toContain('step--current');
+            click(mobileExample + nextStep);
+        }
+    });
+
+    it('should check orientations', () => {
+        wizardPage.checkRtlSwitch();
+    });
+
+    it('should check examples visual regression', () => {
+        wizardPage.saveExampleBaselineScreenshot();
+        expect(wizardPage.compareWithBaseline()).toBeLessThan(5);
+    });
+
+    function checkReOpen(section: string, block: string) {
+        click(section + button);
+        section === defaultExample ? click(block + nextStep) : click(dialogWizard + buttonsBar + ':nth-child(1) ' + button);
+        setValue(fullNameInput, fullName);
+        setValue(firstAdressInput, firstAdress);
+        setValue(secAdressInput, secAdress);
+        section === defaultExample ? click(block + exitButton) : click(dialogWizard + buttonsBar + ':nth-child(3) ' + button);
+        click(section + button);
+        section === defaultExample ? click(block + nextStep) : '';
+        expect(getValue(fullNameInput)).toEqual(fullName);
+        expect(getValue(firstAdressInput)).toEqual(firstAdress);
+        expect(getValue(secAdressInput)).toEqual(secAdress);
+        section === defaultExample ? click(block + exitButton) : click(dialogWizard + buttonsBar + ':nth-child(3) ' + button);
+    }
+
+    function checkOpenClose(section: string) {
+        click(section + button);
+        if (section !== dialogExample) {
+            expect(isElementDisplayed(section + wizard)).toBe(true, `wizard in ${section} did not open`);
+            expect(getAttributeByName(section + wizard, 'style')).toEqual('width: 100%;')
+            click(section + exitButton);
+            expect(isElementDisplayed(section + wizard)).toBe(false, `wizard in ${section} did not close`);
+            expect(getAttributeByName(section + wizard, 'style')).toEqual('width: 0%;');
+        }
+        if (section === dialogExample) {
+            expect(isElementDisplayed(dialogWizard)).toBe(true, 'dialog wizard did not open');
+            click(dialogWizard + buttonsBar + ':nth-child(2) ' + button);
+            expect(doesItExist(dialogWizard)).toBe(false, 'dialog wizard did not close, still exist in DOM');
+        }
+    } 
+
+    function checkStatusStepScroll(section: string, method: 'click' | 'scroll') {
+        click(section + button);
+        const stepsLength = getElementArrayLength(wizard + step);
+        for (let i = 0; i < stepsLength; i++) {
+            click(wizard + step, i);
+            if (i === stepsLength - 1) {
+                method === 'scroll' ? scrollIntoView(contentSection) : '';
```

Comment:
maybe switch is better?
better to only use ternary when you have an action for both cases